runtime/tests/interpreter/resources_test.go

```diff
@@ -2658,205 +2506,6 @@ func TestInterpretDefaultDestroyEventArgumentScoping(t *testing.T) {
 	require.Equal(t, interpreter.NewIntValueFromInt64(nil, 1), events[0].GetField(inter, interpreter.EmptyLocationRange, "x"))
 }
 
-func TestInterpretInnerResourceMove(t *testing.T) {
```

Comment:
Same here: Removed this test since this is no longer valid, after destructor removal